sdv/cag/fixed_increments.py

```diff
@@ -112,12 +111,9 @@ def _validate_constraint_with_data(self, data, metadata):
             data, self._get_single_table_name(metadata), self.column_name, self.increment_value
         )
         if not valid.all():
-            invalid_rows_str = _get_invalid_rows(valid)
-            raise ConstraintNotMetError(
-                'The fixed increments requirement has not been met because the data is not '
-                f"evenly divisible by '{self.increment_value}' for row indices: "
-                f'[{invalid_rows_str}]'
-            )
+            table_name = self._get_single_table_name(metadata)
+            invalid_rows = data[table_name].loc[~valid, [self.column_name]]
+            self._format_error_message_constraint(invalid_rows, table_name)
```

Comment:
Could we leave the `raise ConstraintNotMetError` here and have `_format_error_message_constraint` just return the formatted string? I think it's clearer to have the error message raised here than have it hidden in the helper method. Alternatively, including `raise` in the method name to make it clear that this helper raises the error.